catalogue_pipeline/matcher/src/test/scala/uk/ac/wellcome/platform/matcher/MatcherFeatureTest.scala

```diff
@@ -71,6 +75,52 @@ class MatcherFeatureTest
     }
   }
 
+  it(
+    "does not process a message if the work version is older than that already stored") {
+    withLocalSnsTopic { topic =>
+      withLocalSqsQueue { queue =>
+        withLocalS3Bucket { storageBucket =>
+          withLocalDynamoDbTable { table =>
+            withMatcherServer(queue, storageBucket, topic, table) { _ =>
+              val existingWorkVersion = 2
+              val updatedWorkVersion = 1
+
+              val existingWorkAv2 = WorkNode(
+                id = "sierra-system-number/A",
+                version = existingWorkVersion,
+                linkedIds = Nil,
+                componentId = "sierra-system-number/A"
+              )
+              Scanamo.put(dynamoDbClient)(table.name)(existingWorkAv2)
+
+              val workAv1 = UnidentifiedWork(
+                sourceIdentifier = sourceIdentifierA,
+                identifiers = List(sourceIdentifierA),
+                title = Some("Work"),
+                version = updatedWorkVersion
+              )
+
+              val workSqsMessage: NotificationMessage =
+                hybridRecordNotificationMessage(
+                  message = toJson(RecorderWorkEntry(workAv1)).get,
+                  version = updatedWorkVersion,
+                  s3Client = s3Client,
+                  bucket = storageBucket)
+
+              sqsClient.sendMessage(queue.url, toJson(workSqsMessage).get)
+
+              Thread.sleep(2000)
+              eventually {
+                noMessagesAreWaitingIn(queue)
```

Comment:
you could use the queue and the dlq and check that they are both empty in the eventually (ditching the thread.sleep bit), couldn't you?